vhs-frontend/src/pages/Explore.tsx

```diff
@@ -1,68 +1,51 @@
-import { useEffect, useState } from "react";
-import axios from "axios";
-import { VHS } from "../types";
-import { VhsThumbnail } from "../components/VHSThumbnail";
-import { SearchBar } from "../components/SearchBar";
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { VHS } from '../types';
+import { VhsThumbnail } from '../components/VHSThumbnail';
+import { SearchBar } from '../components/SearchBar';
 // @ts-ignore
-import placeholder from "../assets/placeholder.jpg";
+import placeholder from '../assets/placeholder.jpg';
+import { SearchFormInput } from '../components/searchBarConfig';
 
 export const Explore = () => {
   const [vhsList, setVhsList] = useState<VHS[]>([]);
-  const [filteredList, setFilteredList] = useState<VHS[]>([]);
-  const [searching, setSearching] = useState(false);
+  const [query, setQuery] = useState<SearchFormInput>({ queryType: 'title', queryValue: '' });
 
   useEffect(() => {
-    getVhsList();
-  }, []);
+    getVhsList(query);
+  }, [query]);
 
-  const getVhsList = async () => {
+  const getVhsList = async (query: SearchFormInput) => {
     try {
-      const response = await axios.get("/api/vhs");
+      const response = await axios.get(`/api/vhs`, {
+        params: { [query.queryType]: query.queryValue ? query.queryValue : undefined },
```

Comment:
Instead of ternary operator, try writing `query.queryValue || undefined`. Not sure if it will work but you can try.